src/rpc/blockchain.cpp

```diff
@@ -274,7 +274,7 @@ UniValue waitforblockheight(const JSONRPCRequest& request)
 {
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
         throw runtime_error(
-            "waitforblock\n"
+            "waitforblockheight <blockheight> <timeout>\n"
             "\nWaits for (at least) block height and returns the height and hash\n"
             "\nof the current tip.\n"
```

Comment:
Since you're fixing the comment, we should lose the \n at the start of line 279 so there's only one newline. Otherwise the sentence gets broken up:

```
waitforblockheight <blockheight> <timeout>

Waits for (at least) block height and returns the height and hash

of the current tip.
...
```